conformance/lang/expressions/transactional-expr/transactional_expr.balt

```diff
@@ -0,0 +1,571 @@
+Test-Case: output
+Description: Test whether the transactional expression is being evaluated in transaction mode.
+Labels: check, commit-action, error, optional-type, transaction-stmt, transactional-expr
+
+function init() returns error? {
+    io:println(transactional); // output false
+    transaction {
+        io:println(transactional); // output true
+        check commit;
+        io:println(transactional); // output false
+    }
+    io:println(transactional); // output false
+}
+
+Test-Case: output
+Description: Test whether the transactional expression is being evaluated in transaction mode.
```

Comment:
Shall we not duplicate the description; add a description that specifically describes the scenario